src/main/java/br/com/blackbeard/blackbeardapi/exceptions/FileException.java

```diff
@@ -0,0 +1,8 @@
+package br.com.blackbeard.blackbeardapi.exceptions;
+
+public class FileException extends RuntimeException {
```

Comment:
public class FileException extends RuntimeException {

    public static final String ERROR_TO_READ_FILE = "Error to read file";
    public static final String IMAGE_FORMAT_ERROR_MESSAGE = "The image format must be PNG or JPG";

    public FileException(String msg) {
        super(msg);
    }

    public static FileException errorToReadFile() {
        return new FileException(ERROR_TO_READ_FILE);
    }

    public static FileException invalidImageFormat() {
        return new FileException(IMAGE_FORMAT_ERROR_MESSAGE);
    }
}